frame/democracy/src/vote.rs

```diff
@@ -19,11 +19,12 @@
 
 use sp_std::{prelude::*, result::Result, convert::TryFrom};
 use codec::{Encode, EncodeLike, Decode, Output, Input};
+use scale_info::TypeInfo;
 use sp_runtime::{RuntimeDebug, traits::{Saturating, Zero}};
 use crate::{Conviction, ReferendumIndex, Delegations};
 
 /// A number of lock periods, plus a vote, one way or the other.
-#[derive(Copy, Clone, Eq, PartialEq, Default, RuntimeDebug)]
```

Comment:
yes encoding seems to be a simple byte, with first bit for the `aye` and other bits for the Conviction type.

Reply:
Indeed, I've just described the type as a simple opaque byte. Consumers will have to do further custom decoding to extract the values.